### ISSUE TITLE
Non Critical Enhancement: Clan tier bonus (companion limit increased)

### ISSUE THREAD

**Issue by johnshepardn7117** [Tyler-IN/MnB2-Bannerlord-CommunityPatch#5]
Began as an idea to attempt to get closer to what it was in the last game:
 Suggestion: Doubling or tripling the companion bonus.

If the bonus says +1, to make it more than +1, perhaps a +2 or +3.

Addendum: I tried to make this work with console commands, hence why I am posting it here since I failed. Apparently companions added through console crash the game when used in leadership roles (to be exact, caravan leaders)

**Comment by Tyler-IN** [comment]
Some sort of multiplier, like a difficulty option slider... Ok, sure.

**Comment by KenRiehle** [comment]
I suggest you make these as an optional mod since this is how the game was designed, The original requests are to fix bugs or to enhance the game not make the game easier by lessening the difficulty but this is your mod and your choice, but I want TW to recognize your ability to fix problems.

**Comment by johnshepardn7117** [comment]
> I suggest you make these as an optional mod since this is how the game was designed, The original requests are to fix bugs or to enhance the game not make the game easier by lessening the difficulty ### **_but this is your mod_** and your choice, but I want TW to recognize your ability to fix problems.

mod, modification, alteration of initial function whether it was intended or by mistake. Relax my guy, every group starts the same and ends the same. Tackle errors, followed by improvements.

**Comment by KenRiehle** [comment]
Yeah sorry about that I tend to get a little defense of good work and have
experience with seeing it get derailed and go belly up over going in the
wrong direction. So I do apologize if I said anything wrong.

Thanks for your support.
